src/ansys/fluent/core/workflow.py

```diff
@@ -1060,10 +1251,133 @@ def refresh_after_sleep(_):
+        """
+        self._populate_help_string_command_id_map()
+        if command_name not in self._help_string_command_id_map:
+            raise ValueError(
+                f"'{command_name}' is not an allowed command task.\n"
+                "Use the 'get_possible_tasks()' method to view a list of allowed command tasks."
+            )
+        return self._workflow.InsertNewTask(
+            CommandName=self._help_string_command_id_map[command_name]
+        )
+
+    def delete_tasks(self, list_of_tasks: list[str]):
+        """Delete the list of tasks passed as an argument.
+
+        Parameters
+        ----------
+        list_of_tasks: list[str]
+            List of task items.
+
+        Returns
+        -------
+        None
+
+        Raises
+        ------
+        ValueError
+            If command_name does not match a task name. None of the tasks is deleted.
+        """
+        self._populate_help_string_command_id_map()
+        list_of_tasks_with_display_name = []
+        for task_name in list_of_tasks:
+            try:
+                list_of_tasks_with_display_name.append(
+                    self._help_string_display_text_map[task_name]
+                )
+            except KeyError as ex:
+                raise ValueError(
+                    f"'{task_name}' is not an allowed command task.\n"
+                    "Use the 'get_possible_tasks()' method to view a list of allowed command tasks."
+                ) from ex
+
+        return self._workflow.DeleteTasks(ListOfTasks=list_of_tasks_with_display_name)
+
+    def create_composite_task(self, list_of_tasks: list[str]):
+        """Create the list of tasks passed as argument.
+
+        Parameters
+        ----------
+        list_of_tasks: list[str]
+            List of task items.
+
+        Returns
+        -------
+        None
+
+        Raises
+        ------
+        RuntimeError
+            If the command name does not match a task name.
+        """
+        self._populate_help_string_command_id_map()
+        list_of_tasks_with_display_name = []
+        for task_name in list_of_tasks:
+            try:
+                list_of_tasks_with_display_name.append(
+                    self._help_string_display_text_map[task_name]
+                )
+            except KeyError:
+                raise RuntimeError(
+                    f"'{task_name}' is not an allowed command task.\n"
+                    "Use the 'get_possible_tasks()' method to view a list of allowed command tasks."
+                )
+
+        return self._workflow.CreateCompositeTask(
+            ListOfTasks=list_of_tasks_with_display_name
+        )
+
 
-class OldWorkflowWrapper:
-    """Wrap a Workflow object, adding methods to discover more about the relationships
-    between TaskObjects.
+class ClassicWorkflowWrapper:
+    """Wraps a workflow object, adding methods to discover more about the relationships
+    between task objects.
```

Comment:
**"adding methods to discover more about the relationships between task objects".** 
- That's only true for the EnhancedWorkflowWrapper now, right?
- Now that we have _Enhanced_ and _Classic_ specializations, we can drop the _Wrapper_ part from the ends of the class names. They were only there to differentiate from the wrapped type and that's no longer needed.